frontend/src/views/reading_view.js

```diff
@@ -69,14 +69,17 @@ class ReadingView extends React.Component {
             const questions = QUESTIONS_PROTOTYPE; // replace me with this.state.document.whatever
             const prompts = PROMPTS_PROTOTYPE; // replace me with this.state.document.whatever
             const segment = data.segments[this.state.segmentNum].text;
-
+            const segment_lines = segment.split("\r\n");
             return (
                 <div className={"container"}>
                     <h1>{data.title}</h1>
                     <div className={"row"}>
                         <div className={"col-9"}>
                             <p>Segment Number: {this.state.segmentNum + 1}</p>
-                            <p>{segment}</p>
+                            {segment_lines.map( (line, k) => (
```

Comment:
yay! nitpicks: no space before the (line, k) bit, and get rid of the commented out line below that this code replaces. other than that, ready to merge.

Reply:
@ryaanahmed made the changes!